docker-compose.yml

```diff
@@ -1,17 +1,17 @@
-# version: "3.7"
+version: "3.7"
 
-# services:
-#   web:
-#     build: .
-#     environment:
-#       NODE_ENV:
-#       MONGODB_URI: mongodb://mongo:27017/nodegoat
-#     command: sh -c "until nc -z -w 2 mongo 27017 && echo 'mongo is ready for connections' && node artifacts/db-reset.js && npm start; do sleep 2; done"
-#     ports:
-#       - "4000:4000"
+services:
+  web:
+    build: .
```

Comment:
#### Automated PR Comment From coverity
<!-- 3067c4e8dc6ae8eb83d0a7d419934ecd -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Container allows filesystem write
_Low CWE-552_
The docker service container is configured to permit writing to the root filesystem. This makes some security attack vectors such as privilege escalation, denial-of-service or authorization bypass possible since the container instance's filesystem can be tampered with.
### How to fix
Explicitly set the `read-only` attribute of the service to `true` to create a service container with a read-only filesystem.